website/addons/base/views.py

```diff
@@ -384,6 +384,31 @@ def create_waterbutler_log(payload, **kwargs):
     return {'status': 'success'}
 
 
+@file_signals.file_updated.connect
+def addon_delete_file_node(self, node, event_type, payload, user=None):
+    if event_type == 'file_removed' and payload.get('provider', None) != 'osfstorage':
+        provider = payload['provider']
+        path = payload['metadata']['path']
+        materialized_path = payload['metadata']['materialized']
+        if path.endswith('/'):
+            folder_children = FileNode.resolve_class(provider, FileNode.FILE).find(
+                Q('provider', 'eq', provider) &
+                Q('node', 'eq', node) &
+                Q('materialized_path', 'startswith', materialized_path)
```

Comment:
Yes it does.  The storage root is in the path, not the materialized_path, @felliott, you dummy.